docs/modules/operation/pages/deep-dive/kafka-producer/enable-kafka.adoc

```diff
@@ -1,7 +1,7 @@
 
 = Enable the Kafka Producer
 
-The Kafka Producer is disabled by default and can be enabled as follows.
+The Kafka Producer is disabled by default and can be enabled as follows.  Enabling the Kafka Producer requires that Kafka be installed and running.
```

Comment:
one sentence per line, and it will render as a paragraph on build.
```suggestion
The Kafka Producer is disabled by default and can be enabled as follows.
Enabling the Kafka Producer requires that Kafka be installed and running.
```